cmd/vic-machine/create/create.go

```diff
@@ -864,17 +875,55 @@ func (c *Create) processDNSServers() error {
 	return nil
 }
 
-func (c *Create) processVolumeStores() error {
+func (c *Create) processDefaultVolumeStore() error {
 	defer trace.End(trace.Begin(""))
+	//we make this map here since it is used many times after.
 	c.VolumeLocations = make(map[string]string)
+	if c.DefaultVolumeLocation != "" {
+		c.VolumeLocations[defaultVolumeLabel] = c.DefaultVolumeLocation
+	}
+	return nil
+}
+
+func (c *Create) processVolumeStores() error {
+	defer trace.End(trace.Begin(""))
 	for _, arg := range c.volumeStores {
+
 		splitMeta := strings.SplitN(arg, ":", 2)
 		if len(splitMeta) != 2 {
-			return errors.New("Volume store input must be in format datastore/path:label")
+			return errors.New("Volume store input must be in format datastore:label or datastore/path/to/volumestore:label")
+		}
+		label := splitMeta[1]
+		path := splitMeta[0]
+		var err error
+
+		switch label {
+		case defaultVolumeLabel:
+			customErr := errors.Errorf("multiple paths were tagged for the same label(%s), volumestore labels can only have one distinct path. The preferred method of setting the default store is the --default-volume-store flag.", label)
```

Comment:
"label"?  When did we adopt that verbiage?  Please check the documentation and make sure we're not inventing a new word here.